README.md

```diff
@@ -14,30 +14,183 @@ jobs:
 
     strategy:
       matrix:
-        tutor_version: ["<17.0.0", "==17.0.3", "<18.0.0"]
+        tutor_version: ["<17.0.0", "==17.0.3", "<18.0.0", "nightly"]
```

Comment:
I think this action doesn't work for versions `<17.0.0`. If so, we should mention it and remove these examples. Maybe, also mention that support is only guaranteed for the current version of Open edX, an earlier version, and nightly.

Reply:
yes, it's best to clarify. Thanks!

Reply:
Done